client/my-sites/stats/features/modules/stats-emails/tooltips.tsx

```diff
@@ -0,0 +1,114 @@
+import { Tooltip } from '@automattic/components';
+import { TranslateResult } from 'i18n-calypso';
+import React, { useRef, useState } from 'react';
+
+export interface EmailStatsItem {
+	unique_opens: number;
+	opens: number;
+	opens_rate: number;
+	unique_clicks: number;
+	clicks: number;
+	clicks_rate: number;
+	total_sends: number;
+}
+
+interface TooltipWrapperProps {
+	value: string;
+	item: EmailStatsItem;
+	renderContent: ( item: EmailStatsItem ) => React.ReactNode;
+}
+
+export const TooltipWrapper: React.FC< TooltipWrapperProps > = ( {
+	value,
+	item,
+	renderContent,
+} ) => {
+	const triggerRef = useRef< HTMLSpanElement >( null );
+	const [ showTooltip, setShowTooltip ] = useState( false );
+
+	return (
+		<span className="stats-email__tooltip-wrapper">
+			<span
+				ref={ triggerRef }
+				className="stats-email__tooltip-trigger"
+				onMouseEnter={ () => setShowTooltip( true ) }
+				onMouseLeave={ () => setShowTooltip( false ) }
+			>
+				{ value }
+			</span>
+			<Tooltip position="top" context={ triggerRef.current } isVisible={ showTooltip }>
+				{ renderContent( item ) }
+			</Tooltip>
+		</span>
+	);
+};
+
+export const hasUniqueMetrics = ( uniqueValue: number, totalValue: number ) => {
+	return uniqueValue > 0 && totalValue > 0;
+};
+
+export const createOpensTooltipContent = (
+	item: any,
+	translate: ( text: string, options?: { args: Record< string, any > } ) => TranslateResult
+) => {
+	const opensUnique = parseInt( item.unique_opens, 10 );
+	const opens = parseInt( item.opens, 10 );
+	const opensRate = parseFloat( item.opens_rate );
+	const totalSends = parseInt( item.total_sends, 10 );
+	const hasUniques = hasUniqueMetrics( opensUnique, opens );
+
+	return (
+		<div className="stats-email__tooltip">
+			<div>
+				{ translate( 'Subscribers reached: %(sends)d', {
```

Comment:
Let's use "Recipients" to stay consitent with this copy, which we also just updated:

<img width="298" alt="image" src="https://github.com/user-attachments/assets/60c5e273-9433-45f0-a9db-d9d312ca2581" />